tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -0,0 +1,112 @@
+Test 5-22 - NFS Volume
+=======
+
+# Purpose:
+To verify that NFS shared volumes work with currently supported docker commands
+
+# References:
+[1 - Best practices for running VMware vSphere on NFS](http://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/techpaper/vmware-nfs-bestpractices-white-paper-en.pdf)
+
+[2 - Docker Command Line Reference - Volume Create](https://docs.docker.com/engine/reference/commandline/volume_create/)
+
+[3 - Docker Command Line Reference - Exec](https://docs.docker.com/engine/reference/commandline/exec/)
+
+[4 - Docker Command Line Reference - Volume Inspect](https://docs.docker.com/engine/reference/commandline/volume_inspect/)
+
+[5 - Docker Command Line Reference](https://docs.docker.com/engine/reference/commandline/volume_ls/)
+
+
+# Environment:
+This test requires access to VMware Nimbus for dynamic ESXi and NFS server creation
+
+# Test Steps:
+1. Deploy VIC appliance to an ESX and use a read only NFS mount point
+2. Issue docker volume create on read only volume
+3. Deploy VIC appliance to an ESX and use fake NFS mount point
+4. Deploy VIC appliance to an ESX and use valid NFS mount point
+5. Issue docker volume create using no name for the volume (unnamed) on VolumeStore=nfsVolumeStore (NFS volume)
+6. Run docker mount on container using unnamed volume, run docker wait, then docker rm on temp container
```

Comment:
This can be made clearer with:

```
6. Issue docker run -v using unnamed volume and the mount command, run docker wait, then docker rm on container
```